telephony/android_modem.c

```diff
@@ -1213,6 +1213,23 @@ amodem_send_stk_unsol_proactive_command( AModem  modem, const char* stkCmdPdu )
                           stkCmdPdu); //string type in hexadecimal character format
 }
 
+bool
+amodem_set_hold_feature( bool update, bool enable )
```

Comment:
How about separating the getter to another function, instead of combining it with setter?